backend/requirements.txt

```diff
@@ -0,0 +1,8 @@
+click==6.7
+Flask==0.12.2
+Flask-Cors==3.0.3
+itsdangerous==0.24
+Jinja2==2.9.6
+MarkupSafe==1.0
+six==1.11.0
```

Comment:
This is a nested dependency, should it be listed here? Clearly, we don't need to depend on six since Python 2 is irrelevant.